pubspec.yaml

```
@@ -16,7 +16,7 @@ dependencies:
     sdk: flutter
   package_config: ^2.1.0
   github: ^9.15.0
-  file: ^6.1.4
+  file: ^7.0.0
```

```suggestion
  file: ">=6.1.4 <8.0.0"
```

flutter stable 3.13 unfortunately still depends on file 6.1.4, so specifying `file: ^7.0.0` here would break compatibility with current stable.